tempesta_fw/http.c

```diff
@@ -1866,11 +1884,14 @@ tfw_http_resp_fwd(TfwHttpReq *req, TfwHttpResp *resp)
 		return;
 	}
 	BUG_ON(list_empty(&req->msg.seq_list));
-	req->resp = (TfwHttpMsg *)resp;
+	resp->flags |= TFW_HTTP_RESP_READY;
+	WARN_ON((TfwHttpReq *)resp->req != req);
 	/* Move consecutive requests with @req->resp to @ret_queue. */
 	list_for_each_entry(req, seq_queue, msg.seq_list) {
-		if (req->resp == NULL)
+		if (req->resp == NULL
+		    || !(req->resp->flags & TFW_HTTP_RESP_READY)) {
```

Comment:
It looks to me that either you don't need curly brackets at all as this is a single statement body, or the opening bracket should be on the next line. :-) I think the latter is better. The current variant is no better than no brackets at all.